tests/regression/test_regression.py

```diff
@@ -107,6 +107,21 @@ class TestMultiClassCls(BaseTest):
             extra_overrides={"trainer.max_epochs": "20"},
         )
         for idx in range(1, 4)
+    ] + [
+        DatasetTestCase(
+            name=f"multiclass_CUB_medium",
+            data_root=Path("multiclass_CUB_medium"),
+            data_format="imagenet_with_subset_dirs",
+            num_classes=67,
+            extra_overrides={"trainer.max_epochs": "20"},
```

Comment:
Each small, medium, and large dataset for each task was created with an adjusted volume considering the training time for each task. Is there any particular reason to limit max_epoch to 20 epochs for performance tests? Is it because the purpose of the test is only to see whether regression occurs compared to before?

Reply:
In this phase, the main purpose of sustaining regression test is to check the model performance and iter_time. So, we don't have any problems when we use the fixed epochs.